### DISCUSSION TITLE
Custom pages under Resources folder are not added to left navigation automatically

### DISCUSSION THREAD

**Discussion by sicaboy** [filamentphp/filament#6298] [General]
Only `Filament/Pages/CustonABCPage.php` is added to the left navigation automatically, but not those files under `Filament/Resources/xxResource/Pages/CustonABCPage.php`

Did I miss anything?

**Comment by danharrin** [comment]
Other pages in resources like Create and Edit aren't added either. Check out the `getNavigationItems()` method on the resource class with your IDE. You can override it and add more items for other pages.

**Reply by sicaboy** [reply]
Thanks for answering
